src/components/Events/events.css

```diff
@@ -0,0 +1,40 @@
+.events-row {
+    display: flex;
+    width: 100vw;
+    gap: 1rem;
+    justify-content: center;
+}
+
+.event-card {
```

Comment:
Looks good, we need to remember that will have to make it responsive when we have mobile screen figma.